cloudify_rest_client/deployment_updates.py

```diff
@@ -208,15 +208,14 @@ def update(self,
             params['skip_install'] = skip_install
         if skip_uninstall:
             params['skip_uninstall'] = skip_uninstall
-        if skip_reinstall:
+        if skip_reinstall is not None:
```

Comment:
not really related to the original change, but with just a `if skip_reinstall:` we're not able to pass in a `False`